src/pgsql/pgsql-ddl.lisp

```diff
@@ -306,7 +306,7 @@
 (defmethod format-drop-sql ((fk fkey) &key (stream nil) cascade if-exists)
   (let* ((constraint-name (fkey-name fk))
          (table-name      (format-table-name (fkey-table fk))))
-    (format stream "ALTER TABLE ~a DROP CONSTRAINT~:[~; IF EXISTS~] ~a~@[ CASCADE~];"
+    (format stream "disable_foreign_key_constraints()"
```

Comment:
This'll get called more than once which is suboptimal, but a viable workaround to running with a none-superuser account